script.js

```diff
@@ -1,4 +1,296 @@
+      " <br> <br> Player 1 Optimal Value = " +
+      P1_Max_Combination +
+      "<br> Player 2 Optimal Value = " +
+      P2_Max_Combination +
+      "<br> <br> " +
+      vndCondition +
+      "<br> Record (Player 1 : Player 2) = " +
+      P1_Record +
+      ":" +
+      P2_Record +
+      whoLeads +
+      "<br>Please Click Submit to Play Another Round <br> <br> Refresh to Play Another Mode"
+    );
+  }
+
+  // LCN Game Bundle
+  // Player 1 Bundle (LCN)
+  if (mode == "LCN_GAME") {
+    randomNumber1 = rollDice();
+    randomNumber2 = rollDice();
+    LCN_Final_Number_1 = checkLCN(randomNumber1, randomNumber2);
+
+    mode = "LCN_Player_2";
+
+    return (
+      "Welcome Player 1 <br> You rolled " +
+      randomNumber1 +
+      " for Dice 1 and " +
+      randomNumber2 +
+      " for Dice 2 <br> Player 1 LCN is " +
+      LCN_Final_Number_1 +
+      "<br> It is Player 2's Turn, Please click Submit"
+    );
+  }
+
+  // Player 2 Bundle (LCN)
+  if (mode == "LCN_Player_2") {
+    randomNumber1 = rollDice();
+    randomNumber2 = rollDice();
+    LCN_Final_Number_2 = checkLCN(randomNumber1, randomNumber2);
+    conditionPlease = checkConditionLCN(LCN_Final_Number_1, LCN_Final_Number_2);
+    whoLeads = checkLeading(P1_Record, P2_Record);
+
+    mode = "LCN_GAME";
+
+    return (
+      "Welcome Player 2 <br> You rolled " +
+      randomNumber1 +
+      " for Dice 1 and " +
+      randomNumber2 +
+      " for Dice 2 <br> Player 2 LCN is " +
+      LCN_Final_Number_2 +
+      "<br> Player 1 Generates " +
+      LCN_Final_Number_1 +
+      " Player 2 Generates " +
+      LCN_Final_Number_2 +
+      "<br>" +
+      conditionPlease +
+      "<br> Record (Player 1 : Player 2) = " +
+      P1_Record +
+      ":" +
+      P2_Record +
+      whoLeads +
+      "<br>Please Click Submit to Play Another Round <br> <br> Refresh to Play Another Mode"
+    );
+  }
+
+  // Normal Game Bundle
+  // Player 1 Bundle (NORMAL)
+  if (mode == "NORMAL_GAME") {
+    randomNumber1 = rollDice();
+    randomNumber2 = rollDice();
+
+    mode = "P1_COMBINING_NUMBERS";
+
+    return (
+      "Welcome Player 1 <br> You rolled " +
+      randomNumber1 +
+      " for Dice 1 and " +
+      randomNumber2 +
+      " for Dice 2 <br> choose the order of the dice"
+    );
+  }
+
+  if (mode == "P1_COMBINING_NUMBERS") {
+    if (input == 1) {
+      P1FinalNumber = randomNumber1 * 10 + randomNumber2;
+      myOutputValue =
+        "Player 1, you chose Dice 1 first. <br> Your Number is " +
+        P1FinalNumber +
+        "<br> It is now Player 2's turn. Please click Submit";
+    } else if (input == 2) {
+      P1FinalNumber = randomNumber2 * 10 + randomNumber1;
+      myOutputValue =
+        "Player 1, you chose Dice 2 first. <br> Your Number is " +
+        P1FinalNumber +
+        "<br> It is now Player 2's turn. Please click Submit";
+    }
+    mode = "PLAYER_2_TURN";
+    return myOutputValue;
+  }
+
+  // Player 2 Bundle (NORMAL)
+  if (mode == "PLAYER_2_TURN") {
```

Comment:
With knowledge of arrays and objects, how would you refactor your code, such that you don't have to repeat code for player 2 as you did for player 1? 

Perhaps players within arrays?